src/main/resources/application.properties

```diff
@@ -27,6 +27,9 @@ jira.permission.filepattern=classpath*:permission-templates/jira.permission.*
 
 jira.project.template.key=com.pyxis.greenhopper.jira:gh-scrum-template
 
+# jira notification scheme - attached to the project (10000 is the seed ID for default)
+jira.project.notification.scheme.id=10000
```

Comment:
Wouldn't this need to go into the `ConfigMap` too? I would probably make this to master only?

Reply:
@michaelsauter - this can go to 1.0x as well - it's surgical, and actually also a little bit of a bug .. :)
10000 is the default seed - when you install jira - the chance of ANY reconfig of this is 0 .. that's why I did not include it.

Reply:
OK